clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -249,6 +249,23 @@ bool all(double3);
 _HLSL_BUILTIN_ALIAS(__builtin_hlsl_all)
 bool all(double4);
 
+//===----------------------------------------------------------------------===//
+// and builtins
+//===----------------------------------------------------------------------===//
+
+/// \fn bool and(bool x, bool y)
+/// \brief Logically ands two boolean vectors elementwise and produces a bool
+/// vector output.
+
+_HLSL_BUILTIN_ALIAS(__builtin_hlsl_and)
+bool and (bool x, bool y);
```

Comment:
Weird formatting on these (Possible that clang-format gets confused because it thinks this is C++ and sees a keyword?)

Reply:
Yep thats what is going on here. We should file an issue for this.

Reply:
Filed #127851